Dockerfile

```diff
@@ -70,6 +70,9 @@ RUN apt-get update && apt-get install -y \
     doxygen \
     graphviz
 
+RUN apt-get update && apt-get install -y \
```

Comment:
I'd add a comment above this so it's clear what this is for (even though git blame would reveal it)

Reply:
oh, I forgot that